lua/ulx/modules/sh/cfc_constraint_checker.lua

```diff
@@ -87,38 +94,63 @@ local function countConstraints( plys )
     return perPlyConstraints
 end
 
-local function printConstraintResults( caller, ply, constraintCounts )
+local function printConstraintResults( _, ply, constraintCounts )
```

Comment:
why have a vararg in a function if it's never going to be used?

Reply:
Weird. Opening my code and that's already been addressed, perhaps my code hasn't been fully updated/pushed to GitHub, either way, I already have this cleaned up on my end, and this should be corrected once I'm done with the other tweaks and have them committed and pushed.